wiki/osu!wiki_Contribution_Guide/en.md

```diff
@@ -0,0 +1,102 @@
+At this point, you have two choices:
+
+- [GitHub Web Interface](/wiki/owcg/GitHub_Web_Interface) - edit online; this is best for single article edits
+- [GitHub Desktop](/wiki/owcg/GitHub_Desktop) - edit locally; this is best for single and/or multi article edits (this includes uploading, deleting, and moving images or files)
+
+*Note: You are not limit to the two choices listed above for editing.* There are other third party applications that can do more or less than what the GitHub Web Interface and GitHub Desktop already does.
+
+**When you have completed either *GitHub Web Interface* or *GitHub Desktop*, you can continue to the next section.**
+
+## Finishing
+
+### Opening a pull request
+
+1\. Go to the [`ppy/osu-wiki` repo](https://github.com/ppy/osu-wiki).
+
+2\. If you were quick enough, you may see this yellow banner.
+
+![](img/github-recent.jpg)
+
+3\. If you see it, click on the `Compare & pull request` button (skip to step 8). If not, click the `New pull request` button (continue to step 4).
+
+4\. On the next page, if you see these two buttons as pictured below, click the `compare across forks`.
+
+![](img/compare-across-forks-no.jpg "No good.")
+
+![](img/compare-across-forks-yes.jpg "This is what you want to see.")
+
+5\. Click on the `head fork` dropdown and select the one with your username (it should be the second one).
+
+![](img/head-fork.jpg)
+
+6\. Click on the `compare` dropdown and select the one with the branch you had created (these are listed alphabetically).
+
+![](img/compare-branch.jpg)
+
+7\. Click `Create pull request`.
+
+8\. Enter the title in English. The common title naming scheme is the two-lettered language name in square brackets, followed by the article's title. For example, `[FR] BBCode` would mean it is the French version of the BBCode article.
+
+9\. Fill in the description box with the requested information, you can delete sections if they do not apply.
+
+10\. Once you are ready, click `Create pull request`.
+
+![](img/new-pull-request.jpg "Perfect")
+
+11\. See [Reviews](#reviews) and [Merging](#merging) below.
+
+### Reviews
+
+Once you have created your pull request, other osu!wiki editors may review your changes to help catch some mistakes you may have missed. **You will need to keep up with these reviews,** otherwise your pull request may be marked for closure! If you want someone to review your pull request, you can ask other osu!wiki editors in the [osu!dev Discord chat](https://discord.gg/ppy) or in the GitHub comments.
+
+### Merging
+
+For your changes to become live, your pull request has to be merged. Once your pull request has been reviewed, you can either use the commenting section in GitHub to ask someone to merge it or do the same in the [osu!dev Discord chat](https://discord.gg/ppy).
+
+Once this is completed, your changes are officially part of the osu!wiki. Your changes will not show up right away due to caching on the server's end, but they will show up within one to five hours.
```

Comment:
Where did the 5 hour claim came from? It usually takes up to an hour to appear on the website after a merge.

Reply:
Eh, nevermind.

Got confirmation from peppy that it may take up to 5 hours for changes to appear.

Reply:
Will reword to say that it will take up to five hours to update (and be a little less technical or something)